### ISSUE TITLE
Extension Error: Failed to construct 'URL': Invalid URL

### ISSUE THREAD

**Issue by squaresurf** [GoogleChrome/lighthouse#1209]
**Lighthouse Version**: 1.3.0
**Chrome Version**: 55.0.2883.95
**Error Message**: Failed to construct 'URL': Invalid URL
**Stack Trace**:
 ```TypeError: Failed to construct 'URL': Invalid URL
    at artifacts.DateNowUse.usage.filter.err (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:1046:39)
    at Array.filter (native)
    at Function.audit (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:1045:48)
    at Promise.resolve.then._ (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:11365:20)```

**Comment by wardpeet** [comment]
@squaresurf  could you provide us with an url please.
do you have any chrome plugins installed? Do you have any options deselected on the extension?

// dupe of #1194

**Comment by squaresurf** [comment]
The url is: https://foresight.io/

I have quite a few plugins installed. Is there an easy way to print out the enabled plugins?

All options are selected.

I just tested at this url with a new chrome user with lighthouse as the only plugin and it worked, although there does seem to be an error in the report: `cannot read ts of undefined`
[Lighthouse report_ https___foresight.pdf](https://github.com/GoogleChrome/lighthouse/files/666978/Lighthouse.report_.https___foresight.pdf)

**Comment by ebidel** [comment]
If you go to `chrome://extensions-frame/`, run, this in the console:

```
document.querySelectorAll('.extension-details').forEach(ext => {
  console.log(ext.querySelector('.extension-title').textContent, '-',
                      ext.querySelector('.site-link').href)
});
```

**Comment by oliverturner** [comment]
Just tried in Incognito mode, no extensions enabled bar Lighthouse itself.

<img width="663" alt="screenshot 2016-12-21 18 13 02" src="https://cloud.githubusercontent.com/assets/21795/21400671/2d09d07c-c7a9-11e6-9b56-8350cd05fb2c.png">

**Comment by wardpeet** [comment]
Thanks for the feedback, error should be fixed in #1218

**Comment by squaresurf** [comment]
Great work! Thank you.
